test/base/EtherCollateralLoansWithdrawCollateralTest.js

```diff
@@ -36,15 +35,13 @@ contract('EtherCollateralLoansWithdrawCollateralTest', function (accounts) {
         oracleInstance = await Mock.new();
         loanTermsConsInstance = await Mock.new();
         settingsInstance = await Mock.new();
-        marketsInstance = await Mock.new();
         atmSettingsInstance = await Mock.new();
         instance = await Loans.new();
         await instance.initialize(
             oracleInstance.address,
             lendingPoolInstance.address,
             loanTermsConsInstance.address,
             settingsInstance.address,
-            marketsInstance.address,
             atmSettingsInstance.address,
```

Comment:
The same, I think atmSettingsInstance should be collateralTokenInstance.

Reply:
Changed